src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/AggregationIT.java

```diff
@@ -189,6 +189,30 @@ public void groupByTest() throws Exception {
         Assert.assertThat(gender.query(femaleBucketPrefix + "/COUNT(*)/value"), equalTo(493));
     }
 
+    @Test
+    public void groupByHavingTest() throws Exception {
+
+        JSONObject result = executeQuery(String.format("SELECT gender " +
```

Comment:
Should it be StringUtils.format() vs String.format()?

Reply:
I think internally they have same implementation.